src/util/ScreenSaver/ScreenSaver.h

```diff
@@ -0,0 +1,73 @@
+#include "Cemu/Logging/CemuLogging.h"
+
+#if BOOST_OS_MACOS
+#include <IOKit/pwr_mgt/IOPMLib.h>
+#endif
+
+#if BOOST_OS_LINUX
+#include <SDL2/SDL.h>
+#endif
+
+class ScreenSaver
+{
+public:
+  static void SetInhibit(bool inhibit)
+  {
+    // Adapted from Dolphin's implementation: https://github.com/dolphin-emu/dolphin/blob/e3e6c3dfa41d377520f74ec2488fc1f7b6c05be3/Source/Core/UICommon/UICommon.cpp#L454
+#if BOOST_OS_WINDOWS
```

Comment:
Can't the SDL screensaver inhibitor be used on all platforms? It would simplify code quite a bit.

Reply:
It absolutely can be used on all platforms; after all, that's SDL's ultimate purpose.  :-)  Would still need testing on the other platforms though, just to be sure.  ;-)

Reply:
I was initially hesitant about removing the platform-specific implementations, but now that I think about it again, I'm not exactly sure why.